### PR TITLE
LPS-123195 Add a component that handles the locale changes in the creation of the document types

### PR DESCRIPTION
**Pull request by marcelabc**

### CODE REVIEW

**Comment by marcelabc** [comment]
@marcosapmf

**Comment by marcelabc** [comment]
ci:test:sf

**Comment by marcelabc** [comment]
ci:test:sf

**Comment by marcelabc** [comment]
Hi @oliveiraaraujo ! Could you review my PR, please? :)
Thank you

File: modules/apps/document-library/document-library-web/src/main/resources/META-INF/resources/document_library/js/LocaleChangedHandler.es.js [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+			const inputSelectedValue = inputComponent.getValue(
+				selectedLanguageId
+			);
+
+			if (inputSelectedValue === '') {
+				const inputDefaultValue = inputComponent.getValue(
+					defaultLanguageId
+				);
+
+				// LPS-92493
```
Comment (oliveiraaraujo) [inline modules/apps/document-library/document-library-web/src/main/resources/META-INF/resources/document_library/js/LocaleChangedHandler.es.js:105]: @marcelabc , please remove this commentary.

**Comment by oliveiraaraujo** [comment]
LGTM!

File: modules/apps/document-library/document-library-web/src/main/resources/META-INF/resources/document_library/edit_file_entry_type_data_layout_builder.jsp [hunk 2]
```diff
@@ -93,6 +104,19 @@ renderResponse.setTitle((fileEntryType == null) ? LanguageUtil.get(request, "new
 	</div>
 </aui:form>
 
+<liferay-frontend:component
+	componentId='<%= liferayPortletResponse.getNamespace() + "LocaleChangedHandlerComponent" %>'
+	context='<%=
```
Comment (jeyvison) [inline modules/apps/document-library/document-library-web/src/main/resources/META-INF/resources/document_library/edit_file_entry_type_data_layout_builder.jsp:109]: Can we make a method for this one too? As you did for `dlEditDDMStructureDisplayContext.getComponentContext()`?

**Review by jeyvison — CHANGES_REQUESTED** [review]

File: modules/apps/document-library/document-library-web/src/main/resources/META-INF/resources/document_library/edit_file_entry_type_data_layout_builder.jsp (hunk 2 above)
Comment (marcelabc) [reply modules/apps/document-library/document-library-web/src/main/resources/META-INF/resources/document_library/edit_file_entry_type_data_layout_builder.jsp:109]: Hi @jeyvison 
In edit_ddm_structure_data_layout_builder.jsp, there was already the variable "DLEditDDMStructureDisplayContext" so I added the method in that class.

In this case, there is no var *Context, so I thought of adding directly. Do you think that can maintain this way? Or do you prefer to create the class and add this method?

Comment (jeyvison) [reply modules/apps/document-library/document-library-web/src/main/resources/META-INF/resources/document_library/edit_file_entry_type_data_layout_builder.jsp:109]: Hhhuimm. Got it. No, no need to create the class.

**Review by jeyvison — APPROVED** [review]
LGTM!

File: modules/apps/document-library/document-library-web/src/main/resources/META-INF/resources/document_library/js/LocaleChangedHandler.es.js (hunk 1 above)
Comment (marcelabc) [reply modules/apps/document-library/document-library-web/src/main/resources/META-INF/resources/document_library/js/LocaleChangedHandler.es.js]: Solved!

**Comment by marcelabc** [comment]
ci:test:sf

**Comment by marcelabc** [comment]
ci:test:sf

**Comment by marcelabc** [comment]
ci:forward

**Pull request closed without merging**